report-generator/compile_latency.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+
+LATENCY_ADOC_FILE=$(readlink -f "include/latency-test-reports.adoc")
+STATS_FILE="include/stats_results"
+
+generate_latency_results()
+{
+    # Generate graph and stats associated to a publisher and subscriber test
+    # Parameters: $1: publisher result file
+    # $2: subscriber result file
+
+    local PUB_DATA_FILE=$1
+    local SUB_DATA_FILE=$2
+    python3 timestamp_analysis/timestamp_analysis.py \
+    $PUB_DATA_FILE \
+    $SUB_DATA_FILE >> $STATS_FILE
+}
+
+
+add_latency_results()
+{
+    # Generate and add latency tests results to final report
+    # Parameter: $1: path to tests results directory
+    echo "==  Latency tests results ==" >> "$LATENCY_ADOC_FILE"
+    local TESTS_RESULTS_DIR=$1
+    local SUB_RESULTS_FILES_HYPERVISOR="$(ls "$TESTS_RESULTS_DIR"sub*hypervisor*)" # Getting all sub files results
```

Comment:
Syntax looks weird. Try something simpler:

`SUB_RESULTS_FILES_HYPERVISOR="${TESTS_RESULTS_DIR}/sub*hypervisor*"`

Reply:
Yes, I need to perform a `ls` to list all files with pattern `sub` and `hypervisor`. Without it, it will not work.